package.json

```diff
@@ -116,7 +116,7 @@
     "tslint": "5.14.0",
     "tslint-loader": "3.5.4",
     "tslint-react": "3.6.0",
-    "typescript": "3.4.1",
+    "typescript": "^3.5",
```

Comment:
use specific version instead, 3.5.3, as otherwise yarn.lock will always update in unexpected ways (when upgrading something else)